README.md

```diff
@@ -6,7 +6,7 @@ The bridge is currently implemented in C++ as at the time the Python API for ROS
 Because of this its support is limited to only the message/service types available at compile time of the bridge.
 The bridge provided with the prebuilt ROS 2 binaries includes support for common ROS interfaces (messages/services), such as the interface packages listed in the [ros2/common_interfaces repository](https://github.com/ros2/common_interfaces) and `tf2_msgs`.
 See [the documentation](doc/index.rst) for more details on how ROS 1 and ROS 2 interfaces are associated with each other.
-If you would like to use a bridge with other interfaces (including your own custom types), you will have to build the bridge from source (instructions below).
+If you would like to use a bridge with other interfaces (including your own custom types), you will have to build the bridge from source (instructions below), after building and sourcing your custom types in separate ROS 1 and ROS 2 workspaces. See [the documentation](doc/index.rst) for an example setup.
```

Comment:
Please follow the [style guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#markdown-restructured-text-docblocks): one sentence per line.